include/deal.II/base/aligned_vector.h

```diff
@@ -22,7 +22,15 @@
 #include <deal.II/base/memory_consumption.h>
 #include <deal.II/base/utilities.h>
 #include <deal.II/base/parallel.h>
-#include <boost/serialization/array.hpp>
+
+// boost::serialization::make_array used to be in array.hpp, but was
+// moved to a different file in BOOST 1.64
+#include <boost/version.hpp>
+#if BOOST_VERSION >= 106400
```

Comment:
Don't u need equality here? It was fixed in later commits, so I guess 1.65 won't have this issu